lib/hmmlearn/base.py

```diff
@@ -262,10 +265,10 @@ def _score(self, X, lengths=None, *, compute_posteriors):
         sub_posteriors = [np.empty((0, self.n_components))]
         for sub_X in _utils.split_X_lengths(X, lengths):
             framelogprob = self._compute_log_likelihood(sub_X)
-            logprobij, fwdlattice = self._do_forward_pass(framelogprob)
+            logprobij, fwdlattice = self._do_forward_log_pass(framelogprob)
```

Comment:
why does _score not use the scaling impl. if requested by the user?

Reply:
I've extended scoring to use scaling / log as requested by the user.